### ISSUE TITLE
Switching off LG when room is on temperature

### ISSUE THREAD

**Issue by georgeboot** [altrnate32/esp-lg-control#4]
I am playing with this lib as of today. I do not have an external pump and no external thermostate.

With the relative warm weather of the past days (15 degrees) I can't have the LG run 24/7 because it will get too hot inside.

What is the recommended way of dealing with this? Set the stooklijn low enough so that the LG will pendel or rather make `relay_heat` a real relay to connect it to the LG? The last option is what you use from what I can see.

Sounds to me that is also the best option? And that would also turn off the LG during the summer, instead of just setting a low setpoint and disabling it that way.

Any recommendations?

**Comment by altrnate32** [comment]
Yes you are right, I would recommend an external thermostat (that actuates
relay_heat). You set it to the maximum temperature you want to allow so
that the whole system shuts off. It also helps you save energy when there
is suddenly a lot of sun shingling that heats up your house.

**Comment by georgeboot** [comment]
Thank you!

In your setup you have both `thermostat_signal` and `relay_heat`.

Do I understand it correctly that `thermostat_signal` is the on/off signal from your actual room thermostat? I could easily fake that signal using a temp sensor and an automation.

The control logic then reads that value and based on min run time and other parameters, controls the external thermostat switch on the LG via `relay_heat`, right?

Currently, both of these are template switches in my setup. When I disable `thermostat_signal`, the script will also disable `relay_heat`, but since that is not an actual signal to my LG, the LG never gets any signal to stop making heat.

So bottom line question: is there a way to tell the LG no more heat output is needed WITHOUT actually connecting `relay_heat`?

Only think I can think of, is instead of disabling `thermostat_signal`, just setting the stooklijn offset to -5 or something, but that's probably not ideal.

**Comment by altrnate32** [comment]
Yes you are correct that is exactly how it works. If you currently have
template switches you probably included the without_external_thermostat and
without_external_relays packages. See the readme we if you want this
different.

As far as I know the other way to switch off the LG is through the remote.
You could also lower the setpoint as you say, but then it stays in heating
mode and keeps running the pump. When switches off it is really off (except
frost protection and pump anti stuck)

**Comment by georgeboot** [comment]
Jup did see them and included them as well.

Tried turning off heating via modbus and it seemed to completely shut down.

But sounds like anti freeze and anti stuck will remain active, so that's good enough then. Combined with a 60m pump afterrun, this should be all that I need.

Thanks!

**Comment by Scar172** [comment]
@georgeboot I am struggling with the same situation as you. How did you solve it? Could you maybe share a part of your ESPHome code?

I would like to make an automation with a (virtual) thermostat in HA using the "without_external_thermostat" and "without_external_relays packages".

I can switch off the heat pump with the remote controller or by using the app, however, I would like to do this within Home Assistant so I can create an automation. How did you switch off the HP through modbus? I am curious how you solved this. Thanks in advance!

**Comment by Scar172** [comment]
For other people who are searching how to shutdown there heatpump:
switch:
  - id: heating_switch
    name: "Verwarming On/Off"
    platform: modbus_controller
    modbus_controller_id: lg
    register_type: coil
    address: 0
    # icon: mdi:volume-off

**Comment by georgeboot** [comment]
@Scar172 sorry only see your message now. That is indeed exactly what I ddi!
